src/main/flight/pid.c

```diff
@@ -434,7 +434,7 @@ static float pidLevel(int axis, const pidProfile_t *pidProfile, const rollAndPit
 static float accelerationLimit(int axis, float currentPidSetpoint)
 {
     static float previousSetpoint[3];
-    const float currentVelocity = currentPidSetpoint- previousSetpoint[axis];
+    const float currentVelocity = currentPidSetpoint - previousSetpoint[axis];
```

Comment:
Stray space.

Reply:
Here I just putted the minus in the middle actually

Reply:
Sorry, I looked at the before change. :-|